source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/StakeHolderView.tsx

```diff
@@ -43,21 +43,17 @@ export const StakeHolderView: React.FunctionComponent<IStakeHolderViewProps> = (
     <>
       <StyledSummarySection>
         <LoadingBackdrop show={loading} parentScreen={true} />
-
+        <StyledEditWrapper className="mr-3 my-1">
+          {keycloak.hasClaim(Claims.ACQUISITION_EDIT) && statusSolver.canEditStakeholders() ? (
+            <EditButton title="Edit Interests" onClick={onEdit} />
+          ) : null}
+        </StyledEditWrapper>
         <Section
           isCollapsable
           initiallyExpanded
           header={
             <Row>
-              <Col md="10">Interests</Col>
-              <Col md="2" className="d-flex align-items-center justify-content-end pr-0">
-                <StyledEditWrapper>
-                  {keycloak.hasClaim(Claims.ACQUISITION_EDIT) &&
-                  statusSolver.canEditStakeholders() ? (
-                    <EditButton title="Edit Interests" onClick={onEdit} />
-                  ) : null}
-                </StyledEditWrapper>
-              </Col>
+              <Col>Interests</Col>
             </Row>
           }
```

Comment:
nit: given that we no longer need a "complex" header - we could revert to the simple (string based) header perhaps?
ie: `<Section header="Interests" ...>`